Adding custom rules per our old Rubocop config

senorprogrammer opened the pull request:
These are not necessarily ideal, however they match how the original
config was, and are good enough for the time being.

Commit:
Adding custom rules per our old Rubocop config

These are not necessarily ideal, however they match how the original
config was, and are good enough for the time being.

senorprogrammer committed:
Don't inherit from itself

mrleeio reviewed (commented):
@senorprogrammer excluding stuff is fine.

My concern around providing overrides to rubocop rules is two fold:

1. No one on the team, with the exception of Geoff can tell me why they are enforced
2. Although I can't say this with authority due to it being my 4th week it appears as though people do not update the rules.

If these rule overrides are in place to bring the violation count down thats what a `.rubocop_todo.yml` is for.  The idea being that a developer would uncomment the exclusions in the todo file as they work on that part of the codebase.

All of this is moot if you do have reasons for enabling/disabling/customizing those rules.

Go ahead and merge, make sure you tag it with a new version.  Maybe this is our v1.0?!  The codeship config will need to be updated on all projects that wish to use this.

senorprogrammer commented:
Ah, I like the idea of `.rubocop_todo.yml` - I didn't know about that. That sounds like a good idea.

mrleeio committed:
Update .rubocop.yml

Removed all but one rule override.

Kept line length modifier and altered it to take into account the max character count per line in Github on all major browsers.

[Based on this stackoverflow discussion](https://stackoverflow.com/questions/22207920/what-is-githubs-character-limit-or-line-length-for-viewing-files-on-github)
